mlrun/projects/pipelines.py

```diff
@@ -1074,10 +1075,18 @@ def load_and_run(
         return
 
     # extract "start" notification if exists
+    run_db = mlrun.get_run_db()
+
+    # Here we want to get the run object with the notifications because we want to also get the notifications params
+    # (the context.get_notifications() doesn't include notifications params,
+    # see https://github.com/mlrun/mlrun/pull/4334/files)
```

Comment:
Not a fan of long comments. PR can be added in the description instead.
```suggestion
    # Get the full notifications from the DB since the run context does not contain the params due to bloating
```